index.js

```diff
@@ -31,6 +31,18 @@ function escapeUnsafeChars(unsafeChar) {
     return ESCAPED_CHARS[unsafeChar];
 }
 
+function deleteFunctions(obj){
+    var functionKeys = []
+    for(var key in obj){
```

Comment:
Actually I wrote this first
```javascript
    Object.keys(obj)
    .filter(key => typeof obj[key] === "function")
    .forEach(functionKey => {
      delete obj[functionKey];
    });
```
But I have no idea what the lowest browser the code should support to, 
so I avoid to use `Object.keys`